### PR TITLE
Created function to build Select component options and rectified function names.

### LINKED ISSUE #20: Create a Function buildSelectOption to convert String arrays to Select options object format.

### LINKED ISSUE #21: Change name of function'getColumnDefintion' to 'getColumnDefintions'

### PR DESCRIPTION
**Pull request by eshaanks**
Fixes #20 
Fixes #21 
Had to run some updates hence the yarn.lock file in changes.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 1]
```diff
@@ -10,20 +11,13 @@ const DUMMY_CONTACT_DATA = {
...
-    value: "analyst",
-  },
-];
+export const ROLES = buildSelectOption([
+  "UX designer",
+  "Tester",
+  "Analyst",
+  "Back-end developer",
+  "Front-end developer",
+]);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:20]: Go through my suggestion once again and see if this is the util name I had suggested. Also try to understand why we need to make that name change here.

File: app/javascript/src/utils/index.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+    .map((elem, index) =>
+      index === 0
+        ? elem
+        : elem[0].toUpperCase() + elem.substring(1, elem.length)
+    )
+    .join("");
+
 export const isPresent = R.pipe(R.either(R.isNil, R.isEmpty), R.not);
+
+export const buildSelectOption = optionsAsStringArray =>
```
Comment (yedhink) [inline app/javascript/src/utils/index.js]: https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663250339031239?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

```suggestion
export const buildSelectOptions = values =>
```

**Review by yedhink — CHANGES_REQUESTED** [review]
@eshaanks

File: app/javascript/src/utils/index.js (hunk 2 above)
Comment (eshaanks) [reply app/javascript/src/utils/index.js]: @yedhink  I have made changes accordingly .Please check latest commit https://github.com/eshaanks/neeto-ui-challenge/pull/22/commits/97afceefee0c35b9f10eb95fd2e4696d87b95f7b.

File: app/javascript/src/components/Dashboard/Contacts/constants.js [hunk 3]
```diff
@@ -10,20 +11,13 @@ const DUMMY_CONTACT_DATA = {
...
-  {
-    label: "Tester",
-    value: "tester",
-  },
-  {
-    label: "Analyst",
-    value: "analyst",
-  },
-];
+export const ROLES = buildSelectOption([
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.js:14]: You haven't gone through my suggestion very well. Here we are building `options`. Not a single `option`. Thus plural. Not singular.

So function name should be `buildSelectOptions`.

**Review by yedhink — COMMENTED** [review]
@eshaanks

File: app/javascript/src/components/Dashboard/Contacts/constants.js (hunk 3 above)
Comment (eshaanks) [reply app/javascript/src/components/Dashboard/Contacts/constants.js:14]: Apologies for the oversight. Will fix ASAP

**Pull request merged**